src/node_options.cc

```diff
@@ -777,6 +777,11 @@ PerIsolateOptionsParser::PerIsolateOptionsParser(
   Implies("--experimental-shadow-realm", "--harmony-shadow-realm");
   Implies("--harmony-shadow-realm", "--experimental-shadow-realm");
   ImpliesNot("--no-harmony-shadow-realm", "--experimental-shadow-realm");
+  AddOption("--build-snapshot",
+            "Generate a snapshot blob when the process exits."
+            " Currently only supported in the node_mksnapshot binary.",
```

Comment:
I know this predates the PR but since we are changing it anyway, might as well just remove this line :)

Reply:
Yup, also done in a136c282a9e454929dd6265bf35afefa97591f97 :+1: